modules/maven-plugin-tester/pom.xml

```diff
@@ -25,14 +25,24 @@
                 <groupId>com.smartbear.readyapi</groupId>
                 <artifactId>readyapi4j-maven-plugin</artifactId>
                 <version>${project.version}</version>
-                <configuration>
-                    <username>demoUser</username>
-                    <password>demoPassword</password>
-                    <server>http://testserver.readyapi.io:8080</server>
-                </configuration>
                 <executions>
                     <execution>
-                        <id>run</id>
+                        <id>RunLocally</id>
+                        <configuration>
+                            <runXmlProjects>false</runXmlProjects>
```

Comment:
Yes, we could skip the project execution with a warning message in the logs but then user might think that his tests are working. So, I thought it will be better to fail the build if there are projects to execute with local execution, that will give more clear feedback to the user.